examples/react-redux/js/components/AddTodo.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { PropTypes } from 'react';
+
+export default class AddTodo extends React.Component {
+
+	static propTypes = {
```

Comment:
Removing [the `actions` property](https://github.com/tastejs/todomvc/pull/1706/files#diff-e338e1ac64fad5767aa580c916434ea9R8) throws:

```
Warning: Failed prop type: Required prop `actions` was not specified in `AddTodo`.
    in AddTodo (created by Connect(AddTodo))
    in Connect(AddTodo) (created by Header)
    in header (created by Header)
    in Header (created by App)
    in div (created by App)
    in App (created by Connect(App))
    in Connect(App) (created by RouterContext)
    in RouterContext (created by Router)
    in Router
    in Provider
```

That's because [the `stage-0` preset](https://github.com/tastejs/todomvc/pull/1706/files#diff-640e243774abe1dd121bd12b773b4096R16) contains the [transform-class-properties](https://babeljs.io/docs/plugins/transform-class-properties), transpiling static properties for us.